src/pages/archives.tsx

```diff
@@ -0,0 +1,61 @@
+import { graphql, Link } from "gatsby"
+import * as _ from "lodash"
+import * as React from "react"
+import Layout from "../components/layout"
+import ArchivesSort from "../components/archivesSort"
+
+interface ArchivesPageProps {
+  data: {
+    allMarkdownRemark: {
+      nodes: [
+        {
+          frontmatter: {
+            title: string
+            author: string[]
+            date: string
+          }
+        }
+      ]
+    }
+  }
+}
+
+class CategoriesPage extends React.Component<ArchivesPageProps, {}> {
+  render() {
+    const { data } = this.props
+    const temp = data.allMarkdownRemark.nodes
+
+    return (
+      <div>
+        <Layout>
+          {temp.map((singleNode, index) => (
+            <li key={index}>
```

Comment:
2 updates I would make here: 
- change `Post` to `post` - Capitalized variables generally refer to constructor functions or components.
- instead of `index` pass a unique and stable identifier from inside your `post` object- `post.fields.slug` could be one option (you'll need to add it to the query above)

Reply:
Can we make surer the file-names match the name of the default export exactly (CapsCase, not lower-case) across all files? By having one upper case (where it's used in the code; and then the file name lower-case it creates confusion and drift. If there are any q's about naming just [check out palette docs](https://github.com/artsy/palette/tree/master/packages/palette-docs/src/components). 

(Note you'll have to rename them in git as well, as by default git is case insensitive.)

Reply:
> Can we make sure the file-names match the name of the default export exactly (`CapsCase`, if CapsCase, `lowerCase`, if lowerCase) across all files? By having one upper case (where it's used in the code; and then the file name lower-case it creates confusion and drift. If there are any q's about naming (or anything else) just check out the [conventions set in palette docs](https://github.com/artsy/palette/tree/master/packages/palette-docs/src/components).
> 
> (Note you'll have to rename them in git as well, as by default git is case insensitive.)

Hi, I kept the page files' name in lower case, because the name affects the URL in the archives page. For example, when the archives page URL should be `/archives`. We changed the components name to be capscase.